src/ui/components.qrc

```diff
@@ -3,6 +3,8 @@
         <file>components/forms/qmldir</file>
         <file>components/forms/VPNComboBox.qml</file>
         <file>components/forms/VPNInputBackground.qml</file>
+        <file>components/forms/VPNInputMessage.qml</file>
```

Comment:
`VPNContextualAlert` & `VPNContextualAlerts` maybe? I think the naming here might be confusing since that these don't always appear in inputs.